packages/xo-server/src/xapi/mixins/vm.js

```diff
@@ -262,10 +262,7 @@ export default {
         if (virtualizationMode !== 'pv' && virtualizationMode !== 'hvm') {
           throw new Error(`The virtualization mode must be 'pv' or 'hvm'`)
         }
-        return this._set(
-          'HVM_boot_policy',
-          virtualizationMode === 'hvm' ? 'Boot order' : ''
-        )
+        return this._set('domain_type', virtualizationMode)
```

Comment:
`set_domain_type` is only available starting from XenServer 7.5, we should keep the other one as a fallback:

```js
return this._set('domain_type', virtualizationMode)::pCatch(
  { code: 'MESSAGE_METHOD_UNKNOWN' },
  () =>
    this._set(
      'HVM_boot_policy',
      virtualizationMode === 'hvm' ? 'Boot order' : ''
    )
)
```